packages/did/src/Did.chain.ts

```diff
@@ -153,15 +153,9 @@ function decodeDidPublicKeyDetails(
   const key = keyDetails.key.isPublicEncryptionKey
     ? keyDetails.key.asPublicEncryptionKey
     : keyDetails.key.asPublicVerificationKey
-  const keyType = key.type.toLowerCase() as Lowercase<typeof key.type>
-  if (!keyType) {
-    throw new SDKErrors.DidError(
-      `Unsupported key type "${key.type}" found on chain`
-    )
-  }
   return {
     id: keyId.toHex(),
-    type: keyType,
+    type: key.type.toLowerCase() as typeof key.type,
```

Comment:
that's dealing with the fact that enum type strings are still sentence case, right?

Reply:
Yeah, and might be addressed when we apply a narrower update